README.md

```diff
@@ -10,7 +10,7 @@ You are encouraged to fork this repo and send us pull requests!
 Building
 --------
 
-Install dependencies: ``pip install sphinx sphinx_rtd_theme``.
+Install dependencies: ``pip install sphinx sphinx_rtd_theme sphinx-prompt``.
```

Comment:
I think it would be very convenient to indicate that you need to install sphinx and a link to the project